tests/storage/xfs/test_xfs_sr.py

```diff
@@ -39,6 +40,10 @@ def test_create_and_destroy_sr(self, sr_disk, host_with_xfsprogs):
         vm.destroy(verify=True)
         sr.destroy(verify=True)
 
+@pytest.mark.simple_vdi
+def test_xfs_sr_vdi_is_open(vdi_on_xfs_sr):
+    vdi_is_open(vdi_on_xfs_sr)
+
```

Comment:
Why not make it a method of `TestXFSSR`?

Reply:
Many helpers (Test*SR) use a vm fixture (useless for is_open) like here:
```python
@pytest.mark.usefixtures("nfs_sr", "vm_on_nfs_sr")
class TestNFSSR:
```

I prefer to keep this new test outside these classes.